core/templates/dev/head/dashboard/dashboard.html

```diff
@@ -43,7 +43,7 @@ <h1 class="stat-value"><[dashboardStats.average_ratings || 'N/A']></h1>
           <h1 class="stat-value"><[dashboardStats.total_plays]></h1>
         </div>
         <div class="total-open-feedback stats-card">
-          <p class="stat-description">Open feedback</p>
+          <p class="stat-description">New feedback</p>
```

Comment:
I think it's fine to have "Open Feedback" here and "(new)" below, fwiw. The latter makes me want to click on the row (which is also kind of the point). The former is just informational.